### PR TITLE
Adding devtools button

### PR DESCRIPTION
**Pull request by yaustar**
Devtools from https://yaustar.github.io/playcanvas-devtools/ have now been added.

They can be added to scene via a button in the PlayCanvas panel.

### CODE REVIEW

File: playcanvas-devtools/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
Comment (willeastcott) [inline playcanvas-devtools/LICENSE]: There's already a license in the root - so I guess this can be deleted?

File: playcanvas-devtools/readme.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+![Preview](images/preview-20200715-152807.jpg)
+# How to add to a PlayCanvas App
```
Comment (willeastcott) [inline playcanvas-devtools/readme.md]: Shouldn't all this be integrated into the top level README?

File: playcanvas-devtools/LICENSE (hunk 1 above)
Comment (yaustar) [reply playcanvas-devtools/LICENSE]: Yep, removed.

File: playcanvas-devtools/readme.md (hunk 2 above)
Comment (yaustar) [reply playcanvas-devtools/readme.md]: I was 50/50 on this at the time but have moved it to top level and updated with the new instructions on adding it the app.

**Pull request merged**